README.md

```diff
@@ -63,10 +63,15 @@ If during development the build fails due to `format violations` run the followi
   `deployment/distribution/build/install/samlconf/conf/samlconf-sp-metadata.xml`
   or `samlconf-1.0-SNAPSHOT/conf/samlconf-sp-metadata.xml` from the distribution.
    
-### Implementations
-**TODO** *describe how to create implementations*
+### How to Run Against a Specific Implementation
+This test kit can already be run against [Keycloak](#samlconf-keycloak-impl) and [DDF](#samlconf-ddf-impl) SAML IdP implementations.
+If you're wanting to run the test kit against a different SAML IdP implementation, you will have to follow the below steps.
 
-* Provide your implementations directory to the `samlconf` script using `-i` or `--implementations`.
+* Implement a plugin jar for the implementation
+    * Write a Java or Kotlin class that implements `IdpResponder`. See that interface for details.
+    * Package that file into a jar
+* Place the above jar and the IdP's metadata into a directory. Then that directory should be
+referred to by system property when running tests. (See [Running Test Script](#running_test_script]))
```

Comment:
Just realized I should provide additional steps if they want to contribute their plugin implementation to our project. And strongly encourage that they do so. I will include that in my next commit addressing comments.